stable/sonarqube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.10.3
-appVersion: 6.7.6
+version: 0.10.4
```

Comment:
Please increment the minor version instead as this changes the underlying container version

Reply:
@rjkernick bumping this version is causing issues, as has already been done on your master branch. I have attempted to sort out but not all the sign off requirements seem to be failing. 
I have attempted to correct... but it is still complaining. 

Please can you assist, as I am unaware how to sort this out now, seems a shame to not get this work in cause of version change.